csm_web/frontend/src/components/MentorSection.js

```diff
@@ -296,10 +297,12 @@ function zeroPadTwoDigit(num) {
 class SpacetimeEditModal extends React.Component {
   constructor(props) {
     super(props);
+    // Time string comes as HH:MM:ss, TimeInput expects HH:MM
+    let timeString = props.defaultSpacetime.startTime;
     this.state = {
-      location: "",
-      day: "",
-      time: "",
+      location: props.defaultSpacetime.location,
+      day: props.defaultSpacetime.dayOfWeek,
+      time: timeString.slice(0, timeString.lastIndexOf(":")),
```

Comment:
Can we make this more defensive to make this correct in the case `props.defaultSpacetime.startTime` starts coming in `HH:MM` format in the future?

Reply:
I'd say that if the API becomes HH:MM we should probably be double checking the frontend consumer of the API anyway, but I've added a check